connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/a2a/discovery/A2aGatewayToolHandler.java

```diff
@@ -208,20 +188,26 @@ private ToolCallResult toolCallResultFromA2aSendMessage(ToolCallResult toolCallR
   private String createToolDefinitionDescription(ToolCallResult toolCallResult) {
     try {
       String agentCard = objectMapper.writeValueAsString(toolCallResult.content());
-      return """
-        This tool allows interaction with the remote agent represented by the following agent card:
-        ---
-        %s
-        ---
-        %s
-        """
-          .strip()
-          .formatted(agentCard, TOOL_CALL_RESULT_EXPLANATION);
+      return "This tool allows interaction with the remote agent represented by the following agent card:\n%s"
```

Comment:
💭 Would it make sense to add the term A2A here in the description once again (`this A2A tool`, or `the remote A2A agent`)?

Reply:
Yes :+1: